dashboard/lib/script_constants.rb

```diff
@@ -226,18 +226,9 @@ module ScriptConstants
 
   DEFAULT_VERSION_YEAR = '2017'
 
-  # An allowlist of all family names for scripts.
-  FAMILY_NAMES = [
-    # CSF
-    COURSEA = 'coursea'.freeze,
-    COURSEB = 'courseb'.freeze,
-    COURSEC = 'coursec'.freeze,
-    COURSED = 'coursed'.freeze,
-    COURSEE = 'coursee'.freeze,
-    COURSEF = 'coursef'.freeze,
-    EXPRESS = 'express'.freeze,
-    PREEXPRESS = 'pre-express'.freeze,
```

Comment:
csf family names are still used by _all_courses.html.haml and course_block_helper.rb